### ISSUE TITLE
Cant' load sample data

### ISSUE THREAD

**Issue by xavilondo** [Sylius/Sylius#4799]
Hi,

I'm trying to load sample data on a clean install of sylius but im getting this error:

```
php app/console sylius:install

Installing Sylius...

Step 1 of 4. Checking system requirements.
+--------------------+----------------+
| Issue              | Recommendation |
+--------------------+----------------+
| short_open_tag     |                |
| session.auto_start |                |
+--------------------+----------------+
Success! Your system can run Sylius properly.

Step 2 of 4. Setting up the database.
Creating Sylius database for environment dev.
It appears that your database already exists. Would you like to reset it? (y/N) y
 8/8 [||||||||||||||||||||||||||||] 100%Warning! This will erase your database. Your current environment is dev.
Load sample data? (y/N) y
Loading sample data...
 0/3 [|                           ]   0%
                                                           
  [OutOfBoundsException]                                   
  Reference to: (Sylius.Taxonomy.Category) does not exist  
```

It's maybe any configuration?

**Comment by lchrusciel** [comment]
Have you tried with the newest master? Are you using sylius-sylius(this repo) or sylius-standard? 

I don't have any troubles to load sample data
![zrzut ekranu 2016-04-15 o 13 06 27](https://cloud.githubusercontent.com/assets/6213903/14559717/0929e542-030b-11e6-8139-240b90e92359.png)

**Comment by xavilondo** [comment]
@lchrusciel Im using sylius-standard, how i update to the newest master?

**Comment by michalmarcinkowski** [comment]
@xavilondo change dependency in `composer.json` from `"sylius/sylius": "0.17.*@dev",` to `"sylius/sylius": "0.18.*@dev",` and run `composer update`.

**Comment by xavilondo** [comment]
I updated the dependencies but the error persists.

After a few tries with a clean install with this repo and the standard i detected the bug:

I can't install with this parameter:

`sylius.locale (en_US): es_ES`

With the default value i can install it without problems.

**Comment by a2xchip** [comment]
Seems i had something same as @xavilondo

**Comment by RobertLangore** [comment]
After updating Sylius to latest version, I get this error when running `sylius:install`:
`Step 2 of 4. Setting up the database.
Creating Sylius database for environment dev.
 3/4 [|||||||||||||||||9
 4/4 [|||||||||||||||||9
 4/4 [|||||||||||||||||9
m|||||||||||] 100%
Step 3 of 4. Shop configuration.
Please enter a currency code (For example "GBP") or press ENTER to use "USD".
In which currency can your customers buy goods?
Adding dollaro statunitense


  [RuntimeException]
  Listener "SoftDeleteableListener" was not added to the EventManager!


sylius:install:setup [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-s|--shell] [--process-isolation] [-e|--env ENV] [--no-debug
] [--] <command>`


Is this a bug due to the latest commits?
Can this be solved with some configuration in the "config.yml" file?

**Comment by michalmarcinkowski** [comment]
@RobertLangore can't reproduce that with the newest master, maybe try to clear cache.

**Comment by RobertLangore** [comment]
@michalmarcinkowski Still got the error even after clearing the cache.

From a clean install:
1. composer create-project -s dev sylius/sylius-standard sylius-v018
2. sf cache:clear --no-warmup
3. change "sylius/sylius": "0.17.*@dev" to "sylius/sylius": "0.18.*@dev" in composer.json
4. composer update
5. sf cache:clear --no-warmup
6. sf sylius:install
    then i get exact same error

* sf is an alias of "php app/console"

**Comment by RobertLangore** [comment]
Solved. After the clean install and updating to sylius v0.18 my `app/config/config.yml` was not updated. I had to remove the "filters" in the doctrine configuration:

doctrine:
    orm:
        auto_generate_proxy_classes: "%kernel.debug%"
        entity_managers:
            default:
                auto_mapping: true
                mappings:
                    gedmo_loggable:
                        type: annotation
                        prefix: Gedmo\Loggable\Entity
                        dir: "%kernel.root_dir%/../vendor/gedmo/doctrine-extensions/lib/Gedmo/Loggable/Entity"
                        is_bundle: false
**                filters:
                    softdeleteable:
                        class: Gedmo\SoftDeleteable\Filter\SoftDeleteableFilter
                        enabled: true
**                dql:
                    string_functions:
                        MATCH: Sylius\Bundle\SearchBundle\Extension\Doctrine\MatchAgainstFunction

**Comment by RobertLangore** [comment]
I updated the `app/config/config.yml` in sylius/sylius-standart with the content of the same file in sylius/sylius (after changing the versione from 0.17 in 0.18: as in the previous comment).
@michalmarcinkowski  Are there other files to update?

@xavilondo The same can work for you.

**Comment by michalmarcinkowski** [comment]
Take a look at https://github.com/Sylius/Sylius-Standard/pull/91. I have updated the outdated configuration files on Sylius-Standard.

**Comment by RobertLangore** [comment]
@michalmarcinkowski Thanks.
After updating the files throught git, I got the "Doctrine is not available." error: I solved it with the solution proposed here #4716 (it should be related to the modifications made in symfony-cmf/core-bundle).
